### PR TITLE
Adding poLCA package

### PR DESCRIPTION
**Pull request by Zyg-D**
@conda-forge-admin, please ping team R.
Helper script was used: https://github.com/bgruening/conda_r_skeleton_helper

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Zyg-D** [comment]
@conda-forge-admin, it seems now this can be merged.

**Comment by Zyg-D** [comment]
Hello, @conda-forge-admin, can this be merged?

**Comment by Zyg-D** [comment]
Pinging @conda-forge/help-r

**Comment by Zyg-D** [comment]
Ping @conda-forge/help-r

**Comment by Zyg-D** [comment]
Ping @conda-forge/staged-recipes

**Comment by Zyg-D** [comment]
@cbrueffer - Excuse me. You have just approved and merged my other PR, but I still cannot ping R team myself. Hoping you can help. Thank you.

**Comment by Zyg-D** [comment]
Ping @conda-forge-admin

**Comment by Zyg-D** [comment]
Ping @conda-forge/help-r

File: recipes/r-polca/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  summary: Latent class analysis and latent class regression models for polytomous outcome variables.  Also
+    known as latent structure analysis.
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - dlinzer
```
Comment (dbast) [inline recipes/r-polca/meta.yaml:59]: @Zyg-D who is dlinzer?

Comment (dbast) [reply recipes/r-polca/meta.yaml:59]: Does he agree to be conda-forge package maintainer of the package?

Comment (Zyg-D) [reply recipes/r-polca/meta.yaml:59]: He is the author of the original package. I thought I should reference his name at least. People sometimes do it in their PRs, but myself I haven't contacted him. In this case, should I only leave `conda-forge/r`?

Comment (dbast) [reply recipes/r-polca/meta.yaml:59]: Yes, you can just only add conda-forge/r or also additionally add yourself in case you want to maintain the recipe.

Comment (Zyg-D) [reply recipes/r-polca/meta.yaml]: @dbast - Thank you. I've done what's required.

**Comment by Zyg-D** [comment]
Ping @conda-forge/help-r

**Comment by Zyg-D** [comment]
Hello, @conda-forge/help-r, @conda-forge/staged-recipes, @conda-forge/staged-recipes.  
It seems the PR can be merged now.

**Comment by dbast** [comment]
@Zyg-D thx!

**Pull request merged**